log-warper/src/Log.hs

```diff
@@ -0,0 +1,24 @@
+-- | Reexports all functions from @Log.*@ including @Log.Core@ from
+-- @log-warper-core@ package. See README and tutorials for details.
+
+module Log
```

Comment:
I also think it should be something different. `Logw` or `WLog` would be better imho.

Reply:
@volhovm @gromakovsky O don't have strong preferences of this. I just chose `Log` because it's the shortest one. I like `Logw` more than `WLog` though.